.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+sudo: required
+dist: trusty
+python:
+  - "2.7"
+before_install:
+  - sudo apt-get -qq -y update
+  - sudo apt-get install -y --no-install-recommends zip
+install:
+  - pip install wrc
+script:
+  - git remote add cubing https://github.com/cubing/wca-regulations-translations
+  - git fetch cubing
+  - wget https://github.com/cubing/wca-regulations/archive/official.zip
+  - unzip official.zip
+  - ./travis.sh
```

Comment:
When someone submit a PR we want:

1. to check which translation files have changed
- for each of these files, run the diff against the corresponding one in `official`.
- run the "check" on every translations

For 1. I git-fetched `cubing` because by default travis doesn't seem to do a full clone and fetch remote branches, but it doesn't actually change the content of the current working directory, it just downloads the refs and objects so that I can run the git diff [here](https://github.com/cubing/wca-regulations-translations/pull/35/files#diff-f9a62833538ab14fff849c2be16ab986R3).
2. is made against the `official` archive we just downloaded, and 3. is made in the current working directory (which is, if I'm not mistaken, the content of the PR).

Am I missing something?